pkg/controller/job/job_controller.go

```diff
@@ -72,9 +70,9 @@ var controllerKind = batch.SchemeGroupVersion.WithKind("Job")
 
 var (
 	// DefaultJobBackOff is the default backoff period. Exported for tests.
-	DefaultJobBackOff = 10 * time.Second
+	DefaultJobBackOff = 1 * time.Second
```

Comment:
Does this affect the pod failure backoff as well?
I don't think we should change that

Reply:
Yes, it does. Well, they were coupled so far so seems reasonable this way. I was also thinking that lowering will make the gaps smaller when `RecreationPolicy: Failed`, but I can split into another pair of constants.

Reply:
we should probably decide about pod recreations separately.

Reply:
+1 makes sense. I have split the constants. It is good to probably keep the constant names separately to better reflect what they do.